consensus/scheme/rolldpos/rolldpos.go

```diff
@@ -37,6 +38,32 @@ var (
 	ErrNotEnoughCandidates = errors.New("Candidate pool does not have enough candidates")
 )
 
+// ChainManager defines the blockchain interface
+type ChainManager interface {
```

Comment:
What's the reason of not reusing the existing chain manager interface?

Reply:
Because it has different purpose from existing interface, so the methods in this interface are different.